services/user/user.go

```diff
@@ -74,7 +74,7 @@ func DeleteUser(ctx context.Context, u *user_model.User, purge bool) error {
 					Page:     1,
 				},
 				Private: true,
-				OwnerID: u.ID,
+				Actor:   u,
```

Comment:
```suggestion
				OwnerID: u.ID,
				Actor:   u,
```

Reply:
I know OwnerID is not needed in this case ... but as this "option struct schema" nomaly works it's more future prove that way ...